src/helpers/date.ts

```diff
@@ -0,0 +1,20 @@
+export const formatDate = (date: Date): string => {
```

Comment:
I have a good bet TypeScript on it's own got the typing right here 😁
```suggestion
export const formatDate = (date: Date) => {
```